### PR TITLE
Update cross-tenant-mailbox-migration.md

### PR DESCRIPTION
**Pull request by ropow**
1. Remove references to using -EnableLitigationHoldForMigration to update quotas as the migration will fail if the source mailbox is on hold.

2.  Add section to note that items in the Outbox are client based, and not synced to the cloud, hence migrations of these items is not performed.

### CODE REVIEW

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -406,7 +398,11 @@ While Teams meetings are moved, the meeting URL isn't updated when items migrate
 
 ### What content is migrated cross-tenant?
 
-When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges are migrated.  
+When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges are migrated.
+
+### Do items in the Outbox get migrated cross-tenant?
+
+Items in the Outbox are not migrated cross-tenant as this folder is a client based folder specific to the Outlook client. Items in the Outbox are stored locally, and not synced to the cloud.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
Items in the Outbox are not migrated cross-tenant as this folder is a client-based folder specific to the Outlook client. Items in the Outbox are stored locally, and not synced to the cloud.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ropow, kindly commit the minor suggested change. Many thanks.

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md (hunk 1 above)
Comment (ropow) [reply microsoft-365/enterprise/cross-tenant-mailbox-migration.md]: @JohanFreelancer9 This has been changed and committed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ropow** [comment]
@scotvidican Can you approve this merge, thanks.

**Pull request merged**